users.go

```diff
@@ -101,9 +102,10 @@ type TeamIdentity struct {
 }
 
 type userResponseFull struct {
-	Members      []User                  `json:"members,omitempty"` // ListUsers
-	User         `json:"user,omitempty"` // GetUserInfo
-	UserPresence                         // GetUserPresence
+	Members          []User                  `json:"members,omitempty"` // ListUsers
+	User             `json:"user,omitempty"` // GetUserInfo
+	UserPresence                             // GetUserPresence
+	ResponseMetadata map[string]string       `json:"response_metadata,omitempty"`
```

Comment:
Generally I'm against using maps like this due to how they interact with the heap. use a type for the metadata with the explicit fields you want.